lib/model/message_list.dart

```diff
@@ -222,6 +222,17 @@ mixin _MessageSequence {
     return binarySearchByKey(items, messageId, _compareItemToMessageId);
   }
 
+  Iterable<Message>? getMessagesSublist(int firstMessageId, int lastMessageId) {
```

Comment:
nit: rename to match new semantics:
```suggestion
  Iterable<Message>? getMessagesRange(int firstMessageId, int lastMessageId) {
```